src/elements/content-sidebar/skills/keywords/EditableKeywords.scss

```diff
@@ -10,14 +10,14 @@
         }
     }
 
-    .pill-selector-wrapper {
+    .bdl-PillSelectorDropdown {
```

Comment:
Just noticed that the corresponding file has a "pill-selector-wrapper" classname:  https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/skills/keywords/EditableKeywords.js#L157. Should this classname remain the same?

Reply:
Good catch! Will fix.